frontend/src/data/Site.ts

```diff
@@ -1,19 +1,17 @@
-import URI from "urijs";
-
 const base =
   import.meta.env.MODE === "development"
     ? "https://bowtie.report"
     : window.location.href;
 
-export const siteURI = new URI(base)
-  .fragment("")
-  .directory(import.meta.env.BASE_URL);
+export const siteURI = new URL(base);
+siteURI.hash = "";
+siteURI.pathname = new URL(import.meta.env.BASE_URL, base).pathname;
 
 // FIXME: Presumably a future `Implementation` can handle the below, just
 //        as `Dialect` handles it for dialect reports.
-export const implementationMetadataURI = siteURI
-  .clone()
-  .filename("implementations.json")
-  .href();
+export const implementationMetadataURI = new URL(
+  "implementations.json",
```

Comment:
> any other tests that I should add here @Julian ?

I was going to say "you tell me" :), but considering this line is trivially incorrect, there's at least one here!

Reply:
Wait is it me that's wrong here, does this actually work right? The relative path comes first? I guess I'm not sure myself, so the test couldn't hurt.

Reply:
Yes! The relative path comes first here which makes it really confusing rather having string templates `/` makes it understandable.